packages/roosterjs-editor-dom/lib/selection/Position.ts

```diff
@@ -40,7 +40,7 @@ export default class Position implements NodePosition {
         offsetOrPosType: number = 0,
         private readonly isFromEndOfRange?: boolean
     ) {
-        if ((<NodePosition>nodeOrPosition).node) {
+        if (nodeOrPosition && (<NodePosition>nodeOrPosition).node) {

```

Comment:
This is interesting. In what case you see this is null? I'm afraid there maybe more serious bug hiding here

Reply:
Sometimes, after just hovering the mouse while using cell selection after merging some cells, some node returned null.

Reply:
In that case I'd like to see why that can happen, and fix the root cause, but not just add a simple check here. We already enabled strict mode for this class and the in theory it should never be null.

Reply:
I found the problem in selectTable, in this code 
`table.rows.item(coordinates.firstCell.y).cells.item(coordinates.firstCell.x),`
the the coordinates of the merged cells are being passes, so the it returns null to new Position.
After merge, we need to recalculate the coordinates.

Reply:
I didn't create nothing to recalculate, just added a check to select a cell, if it exist.